update_command.php

```diff
@@ -196,7 +201,7 @@
         // force pushing for cases when doing update-prs
         // double make check we're on a branch that we are willing to force push
         $currentBranch = cmd('git rev-parse --abbrev-ref HEAD', $MODULE_DIR);
-        if (!preg_match('#^pulls/[0-9\.]+/module\-standardiser\-[0-9]{10}$#', $currentBranch)) {
+        if (!preg_match('#^pulls/([0-9\.]+|master|main)/module\-standardiser\-[0-9]{10}$#', $currentBranch)) {
```

Comment:
I could also have changed this to `[a-zA-Z0-9\.]` but I really doubt we'll ever come across a different-named branch, and if we do it's probably best we take a look and make sure things are doing what we expected them to do.